src/app/SecurityPanel/Credentials/StoreJmxCredentials.tsx

```diff
@@ -457,6 +459,14 @@ export const StoreJmxCredentials = () => {
 
 export const StoreJmxCredentialsCard: SecurityCard = {
   title: 'Store JMX Credentials',
-  description: `Credentials that Cryostat uses to connect to target JVMs over JMX are stored here.`,
+  description: (
+    <Text>
+      Credentials that Cryostat uses to connect to target JVMs over JMX are stored here. These are stored in encrypted
+      storage managed by the Cryostat backend. These credentials may be used for manually managing recordings and event
+      templates on target JVMs, as well as for Automated Rules which run in the background and open unattended target
+      connections. Any locally-stored client credentials held by your browser session are not displayed here. See{' '}
+      <Link to="/settings">Settings</Link> to configure locally-stored credentials.
```

Comment:
Maybe, `to configure how/where to store credentials` or something similar because the setting toggles the behaviour?

Reply:
The toggle doesn't disable backend-stored credentials though, it only controls whether the JMX auth modal persists the credentials in the backend or holds them only in browser memory, so I would prefer to keep the "locally-stored" phrasing for this case. The note doesn't really apply to this credentials table, it's just there to explain to the user why they may not see credentials that they entered in the dialog and which they can observe are still present somewhere and usable since they can re-connect to the same target without re-entering credentials within the same browser session.

Reply:
Right! Makes sense! Thanks for explaining that!